src/schema/migrations/migration-5-to-6.sql

```diff
@@ -0,0 +1,4 @@
+set @record_timestamp_index_exists = (SELECT COUNT(1) IndexIsThere FROM INFORMATION_SCHEMA.STATISTICS WHERE table_schema=DATABASE() AND table_name='record' AND index_name='record_timestamp');
```

Comment:
Alter table record to add a column for the record timestamp before adding index.